CHANGELOG.md

```diff
@@ -4,15 +4,53 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.9.26 - Unreleased]
+
+## [0.9.27 - Unreleased]
 ### Added
-- Allow user to remove custom avatar
+
 ### Fixed
 
 ### Changed
-- Improved validation in account recovery. We now show a warning if any words are not from our mnemonic dictionary.
 
-## [0.9.25 - Unreleased]
+## [0.9.26]
+### Added
+- Introduce extended DApp API #5042
+- Webview loading indicator
+- Modal wallet from chat and Dapp browser #5256
+- Auto show existing tokens in wallet #4842
```

Comment:
Auto show existing tokens in wallet *after account recovery*

this only works after recovery, not during the usage